member.go

```diff
@@ -22,11 +22,14 @@ import (
 )
 
 type Member struct {
-	client                   *Client
-	Id                       string   `json:"id"`
-	AvatarHash               string   `json:"avatarHash"`
-	Bio                      string   `json:"bio"`
-	BioData                  string   `json:"bioData"`
+	client     *Client
+	Id         string `json:"id"`
+	AvatarHash string `json:"avatarHash"`
+	Bio        string `json:"bio"`
+	BioData    struct {
+		Emoji struct {
+		} `json:"emoji"`
+	} `json:"bioData"`
```

Comment:
sorry, I guess it's correct --- I was probably confused by an empty Emoji struct. Why is it empty?